invenio/modules/baskets/upgrades/baskets_2015_05_21_groups_module_externalization.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrade script."""
+
+import warnings
+
+from invenio.ext.sqlalchemy import db
+from invenio.modules.upgrader.api import op
+
+
+# Important: Below is only a best guess. You MUST validate which previous
+# upgrade you depend on.
+depends_on = ['groups_2015_04_16_initial']
+
+
+def info():
+    """One line upgrade description."""
+    return "Speparation of groups module changes."
+
+
+def do_upgrade():
+    """Perform upgrade."""
+    if op.has_table('usergroup_bskBASKET'):
+        if op.has_table('group_bskBASKET'):
+            warnings.warn(
+                'Table usergroup_bskBASKET and group_bskBASKET'
+                ' connot exists at the same time!')
+        else:
+            op.rename_table(
+                old_table_name='usergroup_bskBASKET',
+                new_table_name='group_bskBASKET',
+            )
+            op.alter_column(
+                table_name='group_bskBASKET', column_name='id_usergroup',
+                new_column_name='id_group',
+                existing_type=db.Integer(15, unsigned=True),
+                existing_nullable=False, existing_server_default='0',
```

Comment:
If we are doing upgrade recipe I would say we should make it "future" Postgresql friendly ;)